doc/playbook.rst

```diff
@@ -79,66 +87,71 @@ From the ``main`` branch you can release a new major version (``X.0.0``), a new
-* A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
+   For the ``prompt_processing`` service:
 
-For the `next_visit_fan_out`_ service, a new major version is triggered by any of the following:
+   * Incompatibility with old fanned-out ``nextVisit`` messages (almost any change to ``Visit`` qualifies)
+   * Incompatibility with an old `APDB schema`_, `ApdbSql`_, or `ApdbCassandra`_ version (see `DMTN-269`_ for the distinction)
+   * Incompatibility with an old `Butler dimensions-config`_ version
+   * A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
 
-* Incompatibility with old Summit ``nextVisit`` messages
-* Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
+   For the `next_visit_fan_out`_ service:
 
-2. Create a Release
+   * Incompatibility with old Summit ``nextVisit`` messages
+   * Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
 
-On GitHub.com, navigate to the main page of the repository.
-To the right of the list of files, click **Releases**.
-At the top of the page, click **Draft a new release**.
-Type a tag using semantic versioning described in the previous section.
-The Target should be the main branch.
+#. Create a Release
 
-Select **Generate Release Notes**.
-This will generate a list of commit summaries and of submitters.
-Add text as follows.
+   On GitHub.com, navigate to the main page of the repository.
+   To the right of the list of files, click **Releases**.
+   At the top of the page, click **Draft a new release**.
+   Type a tag using semantic versioning described in the previous section.
+   The Target should be the main branch.
 
-* Any specific motivation for the release (for example, including a specific feature, preparing for a specific observing run)
-* Science Pipelines version and rubin-env version
-* Supported `APDB schema`_ and `ApdbSql`_/`ApdbCassandra`_/`ApdbCassandraReplica`_ versions (see `DMTN-269`_ for rationale).
-  A stack quoting a given minor version is compatible with *older* APDBs of that major version but not necessarily newer ones; for example, a release whose baseline is APDB schema 1.4.0 can access a schema 1.0.0 or 1.4.1 database, but not schema 1.5.
-* Supported `Butler dimensions-config`_ versions
-* The `Alerts schema`_ version used for output (see `DMTN-093`_ for details)
+   Select **Generate Release Notes**.
+   This will generate a list of commit summaries and of submitters.
+   Add text as follows:
 
-.. _DMTN-093: https://dmtn-093.lsst.io/#alertmanagement
+   * Any specific motivation for the release (for example, including a specific feature, preparing for a specific observing run)
+   * Science Pipelines version and rubin-env version
+   * Supported `APDB schema`_ and `ApdbSql`_/`ApdbCassandra`_ versions (see `DMTN-269`_ for rationale).
+     You do *not* need to consider the `ApdbCassandraReplica` version.
```

Comment:
Same story -- `ApdbCassandraReplica` does matter, please use the wording on `main`.